src/Field.php

```diff
@@ -33,238 +32,182 @@
 use Yiisoft\Form\Field\Textarea;
 use Yiisoft\Form\Field\Url;
 
-use function array_key_exists;
-
-final class Field
+class Field
```

Comment:
Is the idea to be able to `extend` it in your app to add more shortcuts?

Reply:
Yes. For add shortcuts for non-core fields.